spark-jobs/src/main/scala/za/co/absa/enceladus/spline/persistence/HadoopFsPersistenceFactory.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.spline.persistence
+
+import org.apache.commons.configuration.Configuration
+import org.apache.hadoop.fs.FileSystem
+import org.apache.hadoop.fs.permission.FsPermission
+import org.apache.spark.SparkContext
+import za.co.absa.spline.persistence.api._
+
+/**
+ * The class represents persistence layer factory for Hadoop FS (inluding AWS S3). Based on
+ * * [[za.co.absa.spline.persistence.hdfs.HdfsPersistenceFactory]].
+ */
+object HadoopFsPersistenceFactory {
+  private val fileNameKey = "spline.hdfs.file.name"
```

Comment:
I think this was copied from the old _Spline_, and I'm not sure what naming convention for constants are set in _Enceladus_, but I remember there was a conversation on CQC some time ago - constant names should generally be written uppercase - https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables

Reply:
Usually we adhere to capital letters too. But not very pedantic in code reviews about it. 😉